apps/client/pages/introduction/user-details.tsx

```diff
@@ -0,0 +1,106 @@
+import { useRouter } from 'next/router';
+import { Form, Formik } from 'formik';
+import {
+  Box,
+  Flex,
+  Radio,
+  RadioGroup,
+  Stack,
+  Text,
+  VStack,
+} from '@chakra-ui/react';
+import { useMeQuery, useUpdateUserMutation } from '@agh-kiwis/data-access';
+import { UserPreferencesSchema } from '@agh-kiwis/form-validators';
+import { UserDetailsType } from '@agh-kiwis/types';
+import {
+  CommonButton,
+  CustomSpinner,
+  InputField,
+  Logo,
+  Wrapper,
+} from '@agh-kiwis/ui-components';
+import { initialUserDetails } from '../../formConfig/introductionInitialValues';
+import { mapUserDetailsToUpdateUserMutation } from '../../services/introductionService';
+
+const UserDetails: React.FC = () => {
+  const router = useRouter();
+  const { data, loading } = useMeQuery();
+  const [updateUserMutation] = useUpdateUserMutation();
+
+  const onSubmit = (values: UserDetailsType) => {
+    updateUserMutation({
+      variables: {
+        updateUserInput: mapUserDetailsToUpdateUserMutation(
+          data?.me?.id,
+          values
+        ),
+      },
+    });
+    router.push('/introduction/sleep-preferences');
+  };
+
+  if (loading) {
+    return <CustomSpinner />;
+  }
+  return (
+    <Wrapper>
+      <Logo textVisible={false} />
```

Comment:
Logo should link to the main page

Reply:
I don't agree in this case - this page is one of the introduction steps and I guess that we wouldn't like to send user away to the main page if they accidentally click on it.

Reply:
hmm, makes sense, resolving.